clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/Chat.java

```diff
@@ -27,9 +28,10 @@ public static void main(String[] args) throws Exception {
             });
 
             //This is a blocking call
-            hubConnection.start();
+            hubConnection.startAsync().get();
```

Comment:
Without "awaiting" this call to start we have a race between start and send and the send fails